src/v2/Apps/Auctions/Components/__tests__/AuctionArtworksRails.jest.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { graphql } from "relay-runtime"
+import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
+import { AuctionArtworksRailFragmentContainer } from "../AuctionArtworksRail/AuctionArtworksRail"
+
+jest.unmock("react-relay")
+
+describe("AuctionArtworksRails", () => {
+  const { getWrapper } = setupTestWrapper({
+    Component: (props: any) => {
+      return <AuctionArtworksRailFragmentContainer sale={props.sale} />
+    },
+    query: graphql`
+      query AuctionArtworksRails_Test_Query {
+        sale(id: "xxx") {
+          ...AuctionArtworksRail_sale
+        }
+      }
+    `,
+  })
+
+  it("renders correct components and data", () => {
+    const wrapper = getWrapper({
+      Sale: () => ({
+        internalID: "testid123",
+        href: "/auction/test-href",
+        name: "Test Href",
+        formattedStartDateTime: "Ends Apr 10 at 8:27pm UTC",
+      }),
+    })
+
+    expect(wrapper.find("Waypoint")).toBeDefined()
```

Comment:
I noticed that @damassi used this `toBeDefined` matcher when he did that knowledge share and I was surprised because I always did something more like this:

```ts
expect(wrapper.find("Waypoint")).toHaveLength(1)
```

Any thoughts on a preference here?

Reply:
My thinking is that unless one has some tricky view logic `toBeDefined` seems straight forward enough for a simple component, but I'm down with whatever!

Reply:
I generally prefer the `toBeDefined` approach unless I am attempting to test multiple similar elements on the same page. In which case the usage of numbered instances becomes much more useful.